### PR TITLE
Tweak and add content

### PR DESCRIPTION
**Pull request by ohlincik**
- Add RVM config files to .gitignore
- Optimize the background Bayfront photo
- Add photo and information for Oto
- Add sponsor spot for Tungsten
- Numerous small content tweaks throughout

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,10 @@
 # Ignore bundler config
 /.bundle
 
+# Ignore RVM config files
+.ruby-gemset
+.ruby-version
```
Comment (machuga) [inline .gitignore]: Keeping `.ruby-version` might be a good plan due to the issues you had with 1.9.3.  Can you add one if it doesn't exist?

File: source/index.html.haml [hunk 2]
```diff
@@ -31,19 +31,18 @@ title: Erie Day of Code | Sign up for updates!
     %h2 As a Professional why should I attend?
 
     %p
-      Working with code day in and day out is a fun and intensive task. There are always new tools, languages, or patterns
+      Working with code day in and day out is a fun and intensive undertaking. There are always new tools, languages, or patterns
```
Comment (machuga) [inline source/index.html.haml:34]: :+1:

File: .gitignore (hunk 1 above)
Comment (ohlincik) [inline .gitignore]: Sure. I can do that. In that case we should probably go for 2.1.1 and remove the line from .gitignore?

File: source/index.html.haml [hunk 3]
```diff
@@ -115,13 +114,18 @@ title: Erie Day of Code | Sign up for updates!
                 #{a.address}
             %p.phone= a.phone
 
-    %p If none of these accommodations suit you, please visit #{link_to('Visit PA', 'http://www.visiteriepa.com/where-to-stay')}, or contact #{mail_to('contact', 'contact@eriedayofcode.com')} us to help make arrangements!
+    %p If none of these accommodations suit you, #{link_to('Visit PA', 'http://www.visiteriepa.com/where-to-stay')}, or email #{mail_to('contact', 'contact@eriedayofcode.com')} and we'll help you make arrangements!
```
Comment (machuga) [inline source/index.html.haml]: We need some sort of "click on" or something here for it to read like this is the name of the site "Visit PA"

File: source/layouts/layout.haml [hunk 4]
```diff
@@ -43,6 +43,6 @@
         = yield
         %footer#site-footer
           .contact
-            For more questions or more information, please contact #{mail_to("contact@eriedayofcode.com")}
+            If you have questions or need more information, send email to #{mail_to("contact@eriedayofcode.com")}
```
Comment (machuga) [inline source/layouts/layout.haml]: Can we change "send email" to "send an email"

File: source/index.html.haml (hunk 3 above)
Comment (ohlincik) [inline source/index.html.haml]: I actually thought that it was fun to have it without anything. It reads nice... Visit PA linked is quite obvious IMO.

File: source/layouts/layout.haml (hunk 4 above)
Comment (ohlincik) [inline source/layouts/layout.haml]: Yup, go ahead ... or did you want me to do it? Not sure about the process.

File: speakers.yml [hunk 5]
```diff
@@ -53,8 +53,8 @@
 
 - name: Oto Hlincik
   image: /images/speakers/oto-hlincik.jpg
-  twitter:
-  about: Oto!
-  talk_title: N/A
-  abstract: N/A
-  show: false
+  twitter: "ohlincik"
+  about: Oto is an entrapreneur, developer and overall computer enthusiast. A former hockey player from Slovakia turned web developer and now calling Erie his home. He is currently the Web Division Director at Tungsten Creative Group, helps to organize Erie.rb and lends a hand in organizing the Erie Day of Code. Avid golfer in the Summer, he makes sure that all his web devlopment cravings are satisfied during the infamous Winter Erie months.
```
Comment (machuga) [inline speakers.yml]: Typos: "entrepreneur" and "development"

File: .gitignore (hunk 1 above)
Comment (machuga) [inline .gitignore]: Sure that works for me.

File: source/index.html.haml (hunk 3 above)
Comment (machuga) [inline source/index.html.haml]: I'm not sure everyone will get the reference if I didn't either.

File: source/layouts/layout.haml (hunk 4 above)
Comment (machuga) [inline source/layouts/layout.haml]: If you can, that's typically how I do it, but if you don't have time I can do it as well.

Comment (ohlincik) [inline source/layouts/layout.haml]: Ok, I can do it.

**Comment by machuga** [comment]
Aaaaand merged!

**Pull request merged**